### PR TITLE
AQCU-964: Add discrete WQ data to DV Hydro

### PR DESCRIPTION
**Pull request by zmoore-usgs**

### CODE REVIEW

File: R/dvhydrograph-render.R [hunk 1]
```diff
@@ -293,6 +295,9 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", maxIvLa
     groundWaterLevels = list(
       points = append(list(x=x, y=y, legend.name="Measured Water Level (GWSI)"), styles$gw_level_points)
     ),
+    waterQualityData = list(
```
Comment (thongsav-usgs) [inline R/dvhydrograph-render.R:298]: Will the NULL case break this or was there already a null check somewhere not part of the PR?

Comment (zmoore-usgs) [reply R/dvhydrograph-render.R:298]: I think it should be handled but I'll double check

Comment (zmoore-usgs) [reply R/dvhydrograph-render.R:298]: Yeah NULLs are handled by the plotItem function and if the item to be plotted is null it is skipped and the fetch style function is never called.
